scripts/compare_ci_diff_binables.sh

```diff
@@ -9,10 +9,11 @@ fi
 
 # cleanup if needed
 
-git clean -dfx
+git clean -dfx -e _opam/
```

Comment:
isn't adding `_opam` to .gitignore enough?

Reply:
[`git clean` does not use standard ignore rules when the `-x` switch is set](https://git-scm.com/docs/git-clean#Documentation/git-clean.txt--x).